test/Interop/Cxx/class/access/private-fileid-irgen.swift

```diff
@@ -0,0 +1,114 @@
+//--- blessed.swift
+//
+// RUN: split-file %s %t
+
+// RUN: %target-swift-frontend -emit-ir -module-name main %t/blessed.swift -I %S/Inputs -cxx-interoperability-mode=default | %FileCheck %s
```

Comment:
I am not sure if we need both run lines here. I think the CI will invoke tests both with and without optimizations. We usually have the two run lines when they produce different output in the two modes.

Reply:
Removed the second line

Reply:
(I'm wondering if forcing it to use `-Onone` will resolve the Windows CI issue.)